webapp/components/test-file-results-table.js

```diff
@@ -116,16 +132,29 @@ class TestFileResultsTable extends WPTColors(PathInfo(TestRunsBase)) {
         <td class="sub-test-name"><code>[[ row.name ]]</code></td>
 
         <template is="dom-repeat" items="[[row.results]]" as="result">
-          <td class$="[[ colorClass(result.status) ]]">
-            <code>[[ subtestMessage(result, verbose) ]]</code>
-
-            <template is="dom-if" if="[[result.screenshots]]">
-              <a class="ref-button" href="[[ computeAnalyzerURL(result.screenshots) ]]">
-                <iron-icon icon="image:compare"></iron-icon>
-                COMPARE
-              </a>
-            </template>
-          </td>
+          <template is="dom-if" if="[[ !canAmendMetadata(result.status) ]]">
+            <td class$="[[ colorClass(result.status) ]]">
+              <code>[[ subtestMessage(result, verbose) ]]</code>
+              <template is="dom-if" if="[[result.screenshots]]">
+                <a class="ref-button" href="[[ computeAnalyzerURL(result.screenshots) ]]">
+                  <iron-icon icon="image:compare"></iron-icon>
+                  COMPARE
+                </a>
+              </template>
+            </td>
+          </template>
+
+          <template is="dom-if" if="[[ canAmendMetadata(result.status) ]]">
+            <td class$="[[ colorClass(result.status) ]]" onclick="[[handleSelectMetadata(index, row.name, result.status)]]">
```

Comment:
I know this is how we did it previously, but since this is the second time around I want to take some time to minimize the code duplication.

Is the event handler the only difference between the two branches? We could always bind the event listener and move the branch into the function.